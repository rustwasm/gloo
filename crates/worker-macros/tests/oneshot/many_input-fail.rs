#[gloo::worker::oneshot::oneshot]
async fn Worker(input_1: u32, input_2: u32) -> u32 {
    0
}

fn main() {}
