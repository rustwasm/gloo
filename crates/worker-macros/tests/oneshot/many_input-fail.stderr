error: oneshot worker can accept at most 1 argument
 --> tests/oneshot/many_input-fail.rs:2:31
  |
2 | async fn Worker(input_1: u32, input_2: u32) -> u32 {
  |                               ^^^^^^^^^^^^
